nocodb/Base.py

```diff
@@ -61,8 +61,10 @@ def get_table_by_title(self, title: str) -> Table:
             raise Exception(f"Table with name {title} not found!")
 
     def create_table(self, table_name: str,
-                      columns: list[dict] = [], add_default_columns: bool = True, 
+                      columns: list[dict] = None, add_default_columns: bool = True,
                      **kwargs) -> Table:
+        if columns is None:
+            columns = []
```

Comment:
What is the point of having `None` the default value? And you also have to update the type hint as well.

```suggestion
                      columns: list[dict]|None = None, add_default_columns: bool = True,
                     **kwargs) -> Table:
        if columns is None:
            columns = []
```

Same in the other file. Is it some automatic linting from your editor?

Reply:
VSCode freaks out from this, so we need that `|None`:
![kép](https://github.com/user-attachments/assets/b6c49f4b-1e92-4369-9a2d-ac8194184c43)

Also 2 lines lower there is an `if not columns`, we don't need the `is None`:

```suggestion
                      columns: list[dict] | None = None, add_default_columns: bool = True,
                     **kwargs) -> Table:
          kwargs["table_name"] = table_name
        if not columns:
          kwargs["columns"] = Column.get_id_metadata()
```